Explorer/Assets/DCL/PerformanceAndDiagnostics/Profiling/ECS/DebugViewProfilingSystem.cs

```diff
@@ -125,6 +136,8 @@ void CreateView()
 
         protected override void Update(float t)
         {
+            SampleProfilerCounters();
```

Comment:
I would suggest several things regarding it:
1. It is fully related to the JS scenes, so I would reflect it is in the naming of the function so it is more verbose.
2. No need to collect these counters if respective tab is closed, so I would keep it where `UpdateSceneMetrics()` now.
3. We would like to report this data or part of it to Analytics, so I would move this functionality to our `DCL.Profiling.Profiler` class. Than Analytics can call this function. That also would keep View and Data logic a bit more separate. Analytics would call it rarely (1 time in 15 seconds), so from performance point of view it is ok that same function will be called twice from 2 different Systems (Debug and Analytics).

Reply:
2. I want to see this data in the profiler even when the debug UI tab is closed.

Reply:
Why are there three copies of the Profiler? They are all the same and will collect the same data.
- StaticContainer.cs:170
- FrameTimeCapBudget.cs:38
- MemoryBudget.cs:81

Reply:
Budgets responsibility is to use Profiler data to decide whether to allow or not certain operations to proceed. It relates to execution of certain code parts. While Profiler itself responsible only for data collection and agnostic to execution decisions.

Reply:
Sure, but why three instances?